src/components/overline/index.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Overline from ".";
+
+<Meta title="Components/Overline" component={Overline} />
+
+# Overline
+
+By default, the Overline component will render an overline span for use above headlines or headers. If an href is passed as a property, the Overline will instead render as a simple hyperlink for internal navigation. This is not intended for use for external navigation. If you desire this functionality, please utilize the &lt;Link /&gt; component as a wrap to the Overline.
+
+## Usage
+
+```jsx
+import { Overline } from "@wpmedia/arc-themes-components";
+
+const Feature = () => {
+	return (
+		<>
+			<Overline>Sports</Overline>
+			<h2>Local Team Wins Finals</h2>
```

Comment:
When we are giving example can we be sure not to add confusion to what we are trying to achieve with the components. Using `<h2>` here could add confusion, as it's not something we will do or offer

Reply:
removed